src/test/java/core/basesyntax/model/UserTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.Objects;
+import org.junit.jupiter.api.Test;
+
+class UserTest {
```

Comment:
we don't need to test model in this task

Reply:
remove test class: we don't need to test model in this task